addons/markers/functions/fnc_isMarkerOwner.sqf

```diff
@@ -0,0 +1,30 @@
+#include "script_component.hpp"
+/*
+ * Author: McDiod
+ * Checks whether player created the passed marker.
+ *
+ * Arguments:
+ * 0: Marker <String>
+ *
+ * Return Value:
+ * Whether the player created the marker. <BOOL>
+ *
+ * Example:
+ * [_marker] call ace_markers_fnc_isMarkerOwner
+ *
+ * Public: No
+ */
+
+params [["_marker",""]];
+
+// user has never created a marker
+if (isNil QGVAR(directPlayID)) exitWith {false};
+
+if !((_marker find "_USER_DEFINED") isEqualTo 0) exitWith {false};
+
+private _markerData = (_marker splitString " ") param [1,""];
+private _ownerDirectPlayID = (_markerData splitString "/") param [0,""];
```

Comment:
DirectPlayID? shouldn't this be the playerID aka steamID?

Reply:
See [onPlayerConnected](https://community.bistudio.com/wiki/onPlayerConnected). Apparently this is only used for markers.